src/bentoml/server.py

```diff
@@ -93,14 +93,17 @@ def __init__(
 
     def _create_startmanager(server):  # type: ignore # not calling self self
         class _StartManager:
-            def __init__(self, blocking: bool = False):
+            def __init__(
+                self, blocking: bool = False, env: dict[str, str] | None = None
```

Comment:
should we add a docstring here? users may confuse this with our env manager parameter

Reply:
Done.